jasp-stats/jaspGraphs
# Fix issues with flipped plots for plot editing

Fixes https://github.com/jasp-stats/jasp-test-release/issues/1199

To test this:

1. In jasp-desktop/Modules/jaspMetaAnalysis/DESCRIPTION changes the remote for jaspGraphs into `vandenman/jaspGraphs@forestPlot`
2. build jasp.
3. Follow the steps from https://github.com/jasp-stats/jasp-test-release/issues/1199. Refresh the data library file for meta-analysis and try to edit the forest plot.

Note that I increased the version by 2 as I assume that #28 will be merged first.
